integration-tests/tests/api/schema/contracts-check.spec.ts

```diff
@@ -503,7 +503,7 @@ test.concurrent(
         },
         id: schemaCheckId,
       },
-      authToken: readToken.secret,
+      authToken: ownerToken,
```

Comment:
These operations could be executed via target access tokens. However, it was never intended to be. Now it is only possible with user access tokens.